telegram/ext/_application.py

```diff
@@ -950,7 +963,7 @@ def __run(
 
     def create_task(
         self,
-        coroutine: Union[Generator[Optional["asyncio.Future[object]"], None, RT], Awaitable[RT]],
+        coroutine: _CoroType,
```

Comment:
Here you need to specify the generic `RT` in the definiton of `_CoroType`, i.e. `_CoroType[RT]`. same for other usages of `_(Error)CoroType`. That should then satisfy pyright :)